src/middleware/github-server-app-middleware.ts

```diff
@@ -40,7 +40,7 @@ export const GithubServerAppMiddleware = async (req: Request, res: Response<any,
 		res.locals.gitHubAppId = gitHubServerApp.id;
 		res.locals.gitHubAppConfig = {
 			gitHubAppId: gitHubServerApp.id,
-			appId: gitHubServerApp.appId,
+			appId: String(gitHubServerApp.appId),
```

Comment:
Just wondering about the justification of this change :)

Reply:
The parent object is now more strongly typed.  In other usage it is used as a String this makes this invocation consistent with that typing expectation